packages/thunderstore-api/src/errors.ts

```diff
@@ -7,37 +9,101 @@ type JSONValue =
   | { [x: string]: JSONValue }
   | JSONValue[];
 
+export type ApiErrorContext = {
```

Comment:
(Another example of a type that is used only internally by the file being exported for some reason.)